react/features/display-name/components/web/DisplayNamePrompt.tsx

```diff
@@ -53,10 +61,19 @@ class DisplayNamePrompt extends Component<IProps, IState> {
      * @returns {ReactElement}
      */
     render() {
+        const disableCloseDialog = Boolean(this.props.validateInput);
```

Comment:
I'm not sure why we are tying the disable close to input validation. Can you pl elaborate?

Reply:
Yes and no. For example I think inviting someone to the meeting by phone number would involve validation, but that doesn't mean one shouldn't be able to cancel the operation.

I think input validation and a dialog being cancellable are orthogonal.

Reply:
Ok but in this particular case, if you leave the cancel button there and you have prejoin disabled, you'd be able to skip the requirement for a display name and just attend the meeting. In the prejoin case you'd be prompted for the display name again and again if I drop the state listener and enable the join button.

Reply:
Right. Yeah in this particular dialog it makes sense to tie them together yeah.